zwave_js_server/model/driver.py

```diff
@@ -43,14 +43,14 @@ def handle_all_nodes_ready(self, event: Event) -> None:
 
     async def async_update_log_config(self, log_config: LogConfig) -> bool:
```

Comment:
Should we return `None` implicitly instead then?

Reply:
Yes good idea since we aren't using the return anyway